### PR TITLE
Feature bundle and widths

### PR DESCRIPTION
**Pull request by acuzzio**
Man, do not be an asshole, put this in master without testing. 

Bundleectory

We have renames traj (even in tutorial) and added widths.

### CODE REVIEW

File: aimsprop/fms90.py
Comment (coltonbh) [inline aimsprop/fms90.py:357]: Does this TODO still need to be done before merging?

**Review by coltonbh — DISMISSED** [review]
No tests. As you wish ;)

File: aimsprop/bundle.py
Comment (hayleyweir) [inline aimsprop/bundle.py:29]: widths should be added into the function params comment

File: aimsprop/fms90.py
Comment (hayleyweir) [inline aimsprop/fms90.py:220]: add widths to comment again here

File: aimsprop/fms90.py
Comment (hayleyweir) [inline aimsprop/fms90.py:371]: add FMS.out to this list

File: aimsprop/fms90.py
Comment (hayleyweir) [inline aimsprop/fms90.py:364]: change this so there is no default value and manage error if they do not give a scheme

File: aimsprop/fms90.py
Comment (hayleyweir) [inline aimsprop/fms90.py:406]: could add in print statement saying "applying mulliken /saddle approximation

File: aimsprop/xyz.py
Comment (hayleyweir) [inline aimsprop/xyz.py:18]: should there be an option to input the widths into the function so that parsed xys also have atom widths?

File: docs/tutorial.md
Comment (hayleyweir) [inline docs/tutorial.md:84]: we can add here widths = frame.widths

**Review by hayleyweir — COMMENTED** [review]
looks good. a few changes including updating the comments in the function headers to include widths and adding and example to the tutorial etc. can consider allowing widths to be used when parsing an xyz

File: aimsprop/fms90.py
Comment (acuzzio) [reply aimsprop/fms90.py:357]: We need to wait and talk to Ed. This is not a trivial question.

**Review by coltonbh — COMMENTED** [review]
Leaving comment only. Will leave it to @hayleyweir to approve--like the boss she is. 💁🏻‍♀️

**Review by hayleyweir — APPROVED** [review]
looks good!

**Pull request merged**
